contentcuration/contentcuration/frontend/channelEdit/router.js

```diff
@@ -126,6 +126,48 @@ const router = new VueRouter({
           .then(() => next());
       },
     },
+    {
+      name: RouterNames.TRASH,
+      path: '/:nodeId/:detailNodeId?/trash',
+      component: TrashModal,
+      props: true,
+      beforeEnter: (to, from, next) => {
+        return store
+          .dispatch('currentChannel/loadChannel')
+          .catch(error => {
+            throw new Error(error);
+          })
+          .then(() => next());
```

Comment:
If you'd like to check for edit permissions, you may need to add something like the following as the channel should still load from that endpoint
```
channel => {
  if (channel.edit) {
    next();
  }
}
```

Reply:
I've just checked the whole router file, and we'll need to do it for all routes, and we also don't have the permissions error page, so I think I will leave this as-is for now to not conflict much with user-proofing work.